pkg/roachprod/clusters_cache.go

```diff
@@ -181,14 +181,22 @@ func LoadClusters() error {
 // This function assumes the caller took a lock on a file to ensure that
 // multiple processes don't run through this code at the same time. However, it
 // is allowed for LoadClusters to run in another process at the same time.
-func syncClustersCache(l *logger.Logger, cloud *cloud.Cloud) error {
+//
+// deleteClusters indicates if clusters should be removed if not present in cloud.
```

Comment:
Nit: `dropMissingClusters`? Maybe we could make the param and the comment more precise to reflect that this concerns only the cache replacement policy; i.e., we're only dropping (or not dropping) clusters from cache, whereas `deleteClusters` has a much stronger connotation.

Reply:
Agree with `overwriteMissingClusters`.